synthesizer/process/src/cost.rs

```diff
@@ -395,6 +419,49 @@ pub fn cost_in_microcredits<N: Network>(stack: &Stack<N>, function_name: &Identi
         })
 }
 
+/// Returns the minimum number of microcredits required to run the finalize (depcrated).
+pub fn cost_in_microcredits_deprecated<N: Network>(stack: &Stack<N>, function_name: &Identifier<N>) -> Result<u64> {
```

Comment:
Ditto: Rename `cost_in_microcredits_deprecated` to `cost_in_microcredits_v1` (or something similar)